opslevel/datasource_opslevel_scorecard.go

```diff
@@ -0,0 +1,87 @@
+package opslevel
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func datasourceScorecard() *schema.Resource {
+	return &schema.Resource{
+		Read: wrap(datasourceScorecardRead),
+		Schema: map[string]*schema.Schema{
+			"identifier": {
+				Type:        schema.TypeString,
+				Description: "The id or alias of the scorecard to find.",
+				ForceNew:    true,
+				Optional:    true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The scorecard's name.",
+				ForceNew:    false,
```

Comment:
you can remove the `ForceNew` from everything.  Thats only needed by `resource` schemas